test/SolverInterface.cpp

```diff
@@ -350,6 +353,20 @@ void SolverInterface:: readScalarData
     value = fake_read_write_buffer[0]; 
 }
 
+void SolverInterface:: setMeshAccessRegion
+(
+  int meshID,
+  const double* boundingBox ) const
+{}
+
+void SolverInterface:: getMeshVerticesAndIDs
```

Comment:
```suggestion
void SolverInterface:: getMeshVerticesAndVertexIDs
```
Before it sounded like "get mesh vertices and mesh ids" now it sounds like "get mesh vertices and vertex ids". I think this is what is happening.

Reply:
Okay, I will open an issue in the upstream repo. Lets stick to what we have for now.